packages/entity-list/README.md

```diff
@@ -11,6 +11,7 @@ React-registry name: `entity-list`
 |------------------------------|:---------:|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|-------------------------|
 | `entityName`                 |     *     | Name of the entity that should be listed                                                                                                                                                                | String |                         |
 | `formBase`                   |     *     | formBase_list and formBase_search will be the used forms.                                                                                                                                               | String |                         |
```

Comment:
maybe add .a short sentence for `formBase` as well (will only be used for the search form and not for the list, if `listFormDefinition` is set)